models/articles_model.js

```diff
@@ -3,7 +3,19 @@ const { convertTimestampToDate } = require("../db/helpers/utils.js");
 
 exports.fetchArticleById = (article_id) => {
   return db
-    .query("SELECT * FROM articles WHERE article_id = $1", [article_id])
+    .query("ALTER TABLE articles ADD comment_count INT")
```

Comment:
We do not want to add the comment as a new column, we want to be getting the comment count in a SELECT query. Also this doubles your time and queries made. 

Have a look at how to get your comment count within a SELECT statement, have a look into COUNT(), LEFT JOIN, and GROUP BY as a starting point, and be sure to check the notes app!